### PR TITLE
Makes the Supermatter Announcements More Noticeable.

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
C and B class events, delamination warning announcements, critcal mass/charge inertia/unsafe power level/integrity restored messages and most of the SM detonation countdown are now in bold text.

Class A events, delamination danger announcements, and failsafe deactivation are now in big text.

S class events, 0% integrity, and the final 5 seconds of the detonation countdown are now in megaphone-sized text.

## Why It's Good For The Game
It's harder to accidentally miss SM announcements. If there's a lot going on, an S class event that only gets one warning can be missed. Makes the crystal more scary!

## Images of changes
image
image
image
image

## Testing
Shot the crystal a bunch. Watched the messages.

## Changelog
:cl:
tweak: Supermatter announcements are now generally bigger so they're harder to miss.
/:cl:

### CODE REVIEW

**Review by Contrabang — CHANGES_REQUESTED** [review]
Hallucinations need to be changed also to support this

**Comment by lewcc** [comment]
Is there a sound effect for this currently? I feel like it might make sense to pair these announcements with an alarm so it's less likely for them to disappear silently in chat, like the sound played when a delam is inbound.

**Comment by Fordoxia** [comment]
> Is there a sound effect for these events currently? I feel like it might make sense to pair these announcements with an alarm so it's less likely for them to disappear silently in chat, like the sound played when a delam is inbound.

Most SM events do not have sounds. Only the delamination announcements and the "Oh god it's blowing up" announcement.

These sounds don't actually play except in proximity to the crystal, because it's actually a thing that you can mute the crystal on Telecomms to allow it to silently delaminate if engineering doesn't check up on the crystal.

**Comment by Fordoxia** [comment]
> Hallucinations need to be changed also to support this

The hallucination is now louder.

**Comment by FunnyMan3595** [comment]
Images of the new announcement formatting, please.

**Comment by Fordoxia** [comment]
> Contributor

Done.

File: code/modules/power/engines/supermatter/supermatter.dm [hunk 1]
```diff
@@ -345,9 +345,9 @@
 			sleep(10)
 			continue
 		else if(i > 50)
-			speaking = "[DisplayTimeText(i, TRUE)] remain before causality stabilization."
+			speaking = "<b>[DisplayTimeText(i, TRUE)] remain before causality stabilization.</b>"
 		else
-			speaking = "[i*0.1]..."
+			speaking = "<span class='reallybig'>[i*0.1]...</span>"
```
Comment (DGamerL) [inline code/modules/power/engines/supermatter/supermatter.dm:350]:
```suggestion
			speaking = "<span class='reallybig'>[i * 0.1]...</span>"
```

**Review by DGamerL — APPROVED** [review]
LGTM

**Pull request merged**